integration-tests/fake_spine/fake_spine/config.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+
+class ConfigurationException(Exception):
+    pass
+
+
+def environ_or_error(key: str, default: str = None) -> str:
+    value = os.environ.get(key, default)
+    if not value:
```

Comment:
You should test for None like `if value is None`, as `variable = False` will rise exception